pkgs/development/python-modules/newick/default.nix

```
@@ -0,0 +1,42 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, setuptools-scm
+, pythonOlder
+, pytestCheckHook
+, pytest-cov
+}:
+
+buildPythonPackage rec {
+  pname = "newick";
+  version = "1.9.0";
+  format = "pyproject";
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "dlce-eva";
+    repo = "python-newick";
+    rev = "v1.9.0";
+    sha256 = "TxyR6RYvy2oIcDNZnHrExtPYGspyWOtZqNy488OmWwk=";
+  };
+
+  nativeBuildInputs = [
+    setuptools-scm
+  ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    pytest-cov
+  ];
+
+  pythonImportsCheck = [
+    "newick"
+  ];
+
+  meta = with lib; {
+    description = "python package to read and write the Newick format";
```

```suggestion
    description = "A python package to read and write the Newick format";
```
The first charactor of the description must be capitalized.
https://nixos.org/manual/nixpkgs/stable/#var-meta-description